src/main/java/com/nuclei/assignment/Application.java

```diff
@@ -0,0 +1,88 @@
+package com.nuclei.assignment;
+
+import com.nuclei.assignment.constants.DisplayConstants;
+import com.nuclei.assignment.constants.FileConstants;
+import com.nuclei.assignment.enums.Field;
+import com.nuclei.assignment.enums.Order;
+import com.nuclei.assignment.models.User;
+import com.nuclei.assignment.services.storage.FileStorage;
+import com.nuclei.assignment.services.storage.MemoryImpl;
+import com.nuclei.assignment.services.storage.Storage;
+import com.nuclei.assignment.services.userservice.UserService;
+import java.util.List;
+import java.util.Objects;
+import java.util.Scanner;
+
+public class Application {
+  private Storage file;
+  private Storage memory;
+  private UserService userService;
+  private Scanner sc;
+
+  /**
+   * initialize file and memory storing.
+   */
+  public Application() throws Exception {
+    this.file = new FileStorage(FileConstants.FILE_NAME);
+    this.memory = new MemoryImpl();
+    List<User> users = file.readAll(Field.NAME, Order.ASC);
+
+    if (Objects.nonNull(users) && users.size() != 0) {
+      this.memory.save(users);
+    }
+
+    this.sc = new Scanner(System.in);
+    this.userService = new UserService(sc, memory, file);
```

Comment:
instead of doing this here, create a separate method for initialization. Use a constructor to assign supplied values(through arguments) to fields. Either use another class(factory design pattern) for initialization or create a separate method.